changelog.d/20240108_165452_hrajchert_plt_8891_add_merkleized_contract_flow_example.md

```diff
@@ -0,0 +1,35 @@
+
+### General
+
+- Feat: Added debugging configuration for VSCode. Now if you are developing with VSCode you can open the folder as a workspace and the [Javascript Debug Terminal](https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_javascript-debug-terminal) will have the appropiate source maps. ([PR#136](https://github.com/input-output-hk/marlowe-ts-sdk/pull/136)).
+- Feat: Started an experimental getApplicableActions that should replace the current getApplicableInputs. ([PR#136](https://github.com/input-output-hk/marlowe-ts-sdk/pull/136))
```

Comment:
Are we officially using a convention for the change log ? like Feat, Breaking Change etc ? If yes it should be documented I guess, do we have some documentation around ?

Reply:
we discussed this together with you and @bjornkihlberg.

But yes, we should/could document the structure